desktop-app/app/utils/navigatorUtils.js

```diff
@@ -19,3 +20,28 @@ export function getHomepage() {
 export function getLastOpenedAddress() {
   return settings.get(LAST_OPENED_ADDRESS) || getHomepage();
 }
+
+export function addUrlToExistingSearchResult(existingSearchResults,url){
```

Comment:
@karthick3018 This function cannot be in `urlSearchResultSettings` module as well?